### PR TITLE
Missing logging import

### PR DESCRIPTION
**Pull request by iamamutt**
argparse logging defaults

### CODE REVIEW

File: datajoint_utilities/dj_data_copy/db_migration.py [hunk 1]
```diff
@@ -67,34 +79,47 @@ def migrate_table(orig_tbl, dest_tbl, force_fetch=True):
...
+    is_different_server = (
+        orig_tbl.connection.conn_info["host"] != dest_tbl.connection.conn_info["host"]
+    )
 
     # check if there's external datatype to be transferred
-    has_external = np.any(['@' in attr.type
-                           for attr in orig_tbl.heading.attributes.values()])
+    has_external = np.any(
+        ["@" in attr.type for attr in orig_tbl.heading.attributes.values()]
+    )
```
Comment (dimitri-yatsenko) [inline datajoint_utilities/dj_data_copy/db_migration.py:99]:
```suggestion
    has_external = any("@" in attr.type for attr in orig_tbl.heading.attributes.values())
```

File: datajoint_utilities/dj_worker/__init__.py [hunk 2]
```diff
@@ -153,28 +183,40 @@ def _clean_up(pipeline_modules, additional_error_patterns=[], stale_hours=24):
...
+            & [
+                f'error_message LIKE "{e}"'
+                for e in _generic_errors + additional_error_patterns
+            ]
+        ).delete()
         # clear stale "reserved" jobs
-        stale_jobs = ((pipeline_module.schema.jobs & 'status = "reserved"').proj(
-            elapsed_days='TIMESTAMPDIFF(HOUR, timestamp, NOW())')
-                      & f'elapsed_days > {stale_hours}')
+        stale_jobs = (pipeline_module.schema.jobs & 'status = "reserved"').proj(
```
Comment (dimitri-yatsenko) [inline datajoint_utilities/dj_worker/__init__.py:208]: There is a better way to do this. The jobs table contains the connection number. A stale job is one for which there is no connection anymore.

**Review by dimitri-yatsenko — CHANGES_REQUESTED** [review]
There is no array work, so no need to import and use numpy.

**Comment by ttngu207** [comment]
@dimitri-yatsenko we'll address those 2 points in the next PR. I'm merging this now as it is needed for aeon ingestion work

**Review by ttngu207 — APPROVED** [review]

File: datajoint_utilities/dj_worker/__init__.py (hunk 2 above)
Comment (dimitri-yatsenko) [reply datajoint_utilities/dj_worker/__init__.py:208]: Also, you need to disable serialization when moving data.  You will save a lot of time and errors.  `dj.blob.bypass_serialization = True`.

**Pull request merged**